.werft/util/werft.ts

```diff
@@ -86,27 +108,48 @@ export class Werft {
     }
 
     public done(slice: string) {
-        const span = this.sliceSpans[slice]
-        if (span) {
-            span.end()
-            delete this.sliceSpans[slice]
+
+        // This is only for backwards compatibility!
+        if (!this.phases[this.currentPhase].slices[slice]) {
+            this.newSlice(slice);
```

Comment:
Can you elaboarate the use case that we want to be beackwards compatible with? 
It feels wrong the create a slice here when we're actualy done.
Better: Log a warning that the code attemts to close a slice that had never been opened.

Reply:
I added a warning.